src/main/resources/default/templates/biz/process/process.html.pasta

```diff
@@ -23,6 +22,17 @@
                 </span>
             </i:block>
             <i:block name="actions">
+                <i:if test="process.shouldAutorefresh()">
+                    <a id="disableAutoRefresh"
+                       class="btn btn-link d-none"
+                       href="#">
```

Comment:
```suggestion
                       href="javascript:enableAutoRefresh()">
```

Reply:
less magic...more obvious (or via click="...")

Reply:
Disadvantage: You have to put `enableAutoRefresh()` in the global js context